run.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
-cd llama.cpp
-make llama-server
-cd ..
 
-export LLAMA_N_CTX=32000
+if nvcc --version
```

Comment:
Doesn't this fail if you don't have the binary?

Reply:
yeah, and it goes on the `else`